docs/book/src/clusterctl/overview.md

```diff
@@ -100,10 +100,10 @@ See the [AWS Provider Prerequisites](https://github.com/kubernetes-sigs/cluster-
 
 ```bash
 # Create the base64 encoded credentials
-export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 | tr -d '\n')"
-export AZURE_TENANT_ID_B64="$(echo -n "$AZURE_TENANT_ID" | base64 | tr -d '\n')"
-export AZURE_CLIENT_ID_B64="$(echo -n "$AZURE_CLIENT_ID" | base64 | tr -d '\n')"
-export AZURE_CLIENT_SECRET_B64="$(echo -n "$AZURE_CLIENT_SECRET" | base64 | tr -d '\n')"
+export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 -w0)"
```

Comment:
Unfortunately this doesn't work on Mac, and this is the best way we found to work across operating systems without installing extra dependencies.

Reply:
Ah ok, closing then :)